src/main/java/com/synopsys/integration/alert/channel/azure/boards/service/AzureBoardsProperties.java

```diff
@@ -108,6 +110,15 @@ public String getOauthUserId() {
         return scopes;
     }
 
+    public void validateProperties() throws AlertConfigurationException {
+        if (StringUtils.isBlank(organizationName) || StringUtils.isBlank(clientId) || StringUtils.isBlank(clientSecret)) {
+            throw new AlertConfigurationException("The global configuration for Azure is missing required information.");
+        }
+        if (StringUtils.isBlank(oauthUserId)) {
```

Comment:
The `oauthUserId` doesn't get populated until the authentication with Azure occurs, right?

Reply:
The `oauthUserId` get's populated when `fromFieldAccessor` is called.  We don't know the user when performing the handshake so we are always using the value of `DEFAULT_AZURE_OAUTH_USER_ID` constant.  Are you seeing it have a value other than the default?

Reply:
Ok. I didn't realize that we were setting a default value for that field. I thought it would only get populated if the User has pressed the "Authenticate" button in  the Azure configuration. I assumed it would be blank if the User filled in the fields and only pressed save not authenticate. 
If this field is always going to have a value then should I remove this part of the validation?

Reply:
yes I would remove it.  The documentation has an example where the email address is passed in via the state query parameter.  We don't capture the user email in the global configuration from the UI to be able to include it in the state query parameter.  Since we have only one configuration for Azure boards our default key is sufficient for now.  If we started having multiple Azure boards tokens then we would need unique keys.